po/r-novice-gapminder.ja.po

```diff
@@ -717,14 +717,13 @@ msgid ""
 "practices for scientific computing: breaking down analyses into modular units,\n"
 "task automation, and encapsulation."
 msgstr ""
-"The goal of this lesson is to teach novice programmers to write modular code\n"
-"and best practices for using R for data analysis. R is commonly used in many\n"
-"scientific disciplines for statistical analysis and its array of third-party\n"
-"packages. We find that many scientists who come to Software Carpentry workshops\n"
-"use R and want to learn more. The emphasis of these materials is to give\n"
-"attendees a strong foundation in the fundamentals of R, and to teach best\n"
-"practices for scientific computing: breaking down analyses into modular units,\n"
-"task automation, and encapsulation."
+"この講座の目的は、プログラム初心者に R を使ったデータ解析のためのモジュラーコードの書き方と、\n"
+"そのベストプラクティスを教えることです。R は多くの科学分野において、\n"
+"統計分析とそれに伴うサードパーティーパッケージに使用されています。\n"
```

Comment:
This isn't quite accurate. The original is saying that the reason R is used is because it has a wide variety of third-party packages, not that R is used by third-party packages.

Reply:
@masamiy can you take a look at this? Since it's your branch I'm not able to accept suggested changes. Otherwise I'd merge the one above since Joel and Kozo agree on it.

Reply:
What about this:
”この講座の目的は、プログラミング初心者に R を使ったデータ解析のためのモジュール化されたコードの書き方と、
そのベストプラクティスを教えることです。R には多くの科学分野の統計分析に使用できるサードパーティパッケージが用意されています。

Reply:
Close! How about this for the last sentence:
R は多くの科学分野に統計分析やサードパーティパッケージによる様々なデータ解析に使われています。
Unfortunately the original English is poorly written and ambiguous,  but I think this is closer to the original intent.

You can update this PR by making another commit locally and pushing it (I can help in slack or zoom if needed).

Reply:
I prefer 'の’ instead of 'に’, i.e.,
R は多くの科学分野**の**統計分析やサードパーティパッケージによる様々なデータ解析に使われています。

If it is okay, then, I see if I can commit by myself. I will ask your remote support next week if needed. Thank you for the offer as always.

Reply:
Sounds good to me. Just to clarify: to make the change, do exactly what you did before (edit the file, commit the change, push the branch to your fork, then make a pull request). Let me know how it goes!

Reply:
Seems to be on the right track here. The English is ambiguous here, it means: R is used for the purpose of scientific analysis and to make use of many 3rd party packages. As long as we cover that intention I think it's fine. I've opened a PR here attempting to make these changes but I think what's discussed above is better. We can close my PR if Masami can make the changes to `intro-edit` branch locally.
 https://github.com/masamiy/i18n/pull/3/files